vsd-app/src/api/Startup.cs

```diff
@@ -134,43 +134,9 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
                 app.UseExceptionHandler("/Home/Error");
             }
 
-            app.Use(async (ctx, next) =>
-            {
-                ctx.Response.Headers.Add("Content-Security-Policy",
-                                         "script-src 'self' 'unsafe-eval' 'unsafe-inline' https://apis.google.com https://maxcdn.bootstrapcdn.com https://cdnjs.cloudflare.com https://code.jquery.com https://stackpath.bootstrapcdn.com https://fonts.googleapis.com");
-                ctx.Response.Headers.Add("Strict-Transport-Security", "max-age=31536000; includeSubDomains; preload");
-                await next();
-            });
             app.UseXContentTypeOptions();
             app.UseXfo(xfo => xfo.Deny());
 
-            if (!env.IsDevelopment())  // when running locally we can't have a strict CSP
```

Comment:
This was excluded from development since it imposes a strict CSP. I have since replaced this instead with the CSP being implemented inside of the `Caddyfile`. Please let me know if you think this is sufficient or not.